java/src/jmri/jmrit/logix/TrackerTableAction.java

```diff
@@ -392,254 +724,6 @@ public void windowClosing(java.awt.event.WindowEvent e) {
-                    block.addPropertyChangeListener(this);
-                    if (log.isDebugEnabled()) {
-                        log.debug("\taddPropertyChangeListener for block {}", block.getDisplayName());
-                    }
-                } else {
-                    if (log.isDebugEnabled()) {
-                        log.debug("\tassumed block {} already has listener" + block.getDisplayName());
-                    }
-                }
-                if (!trackers.contains(tracker)) {
-                    trackers.add(tracker);
-                }
-            }
-        }
-
-        /**
-         * Do Venn Diagram between the two sets. Keep listeners held in common.
-         * Add new listeners. Remove old.
-         */
-        private void adjustBlockListeners(List<OBlock> oldRange, List<OBlock> newRange, Tracker tracker) {
-            Iterator<OBlock> iter = newRange.iterator();
-            while (iter.hasNext()) {
-                OBlock b = iter.next();
-                if (oldRange.contains(b)) {
-                    oldRange.remove(b);
-                    continue; // held in common. keep listener
-                }
-                addBlockListener(b, tracker);       // new block.  Add Listener
-            }
-            // blocks left in oldRange were not found in newRange.  Remove Listeners
-            iter = oldRange.iterator();
-            while (iter.hasNext()) {
-                removeBlockListener(iter.next(), tracker);
-            }
-
-        }
-
-        private void removeBlockListeners(List<OBlock> range, Tracker tracker) {
-            if (log.isDebugEnabled()) {
-                log.debug("removeBlockListeners for tracker= \"" + tracker.getTrainName()
-                        + "\" has " + range.size() + " blocks to remove.");
-            }
-            Iterator<OBlock> iter = range.iterator();
-            while (iter.hasNext()) {
-                removeBlockListener(iter.next(), tracker);
-            }
-        }
-
-        private void removeBlockListener(OBlock b, Tracker tracker) {
-            List<Tracker> trackers = _blocks.get(b);
-            if (trackers != null) {
-                trackers.remove(tracker);
-                if (trackers.size() == 0) {
-                    b.removePropertyChangeListener(this);
-                    if (log.isDebugEnabled()) {
-                        log.debug("removeBlockListener on block " + b.getDisplayName()
-                                + " for tracker= " + tracker.getTrainName());
-                    }
-                }
-            } else {
-                if (log.isDebugEnabled()) {
-                    log.error("Block \"" + b.getDisplayName() + "\" has no listeners.  Tracker for train "
-                            + tracker.getTrainName() + " expected a listener");
-                }
-            }
-        }
-
-        @Override
-        public void propertyChange(java.beans.PropertyChangeEvent evt) {
```

Comment:
This also looks like a public interface method being removed.  Or has it just been moved?